routes/event_router.py

```diff
@@ -41,7 +41,7 @@ def update(event_id: int, data: EventUpdate, db: DB_dependency):
 def getAllSignups(event_id: int, db: DB_dependency):
     peoplesignups = db.query(EventUser_DB).filter_by(event_id = event_id).all()
     if len(peoplesignups)==0:
-        return []

```

Comment:
Here I think it is more readable if you put `users: list[] = []` above the if and return `users`  instead of `list[User_DB]`. It looks like a weird way to create an empty list, or maybe it returns the constructor?